modules/boot/src/utils/link.cpp

```diff
@@ -78,6 +80,31 @@ KEEP_FUNC void GZ_displayTimeInfo() {
                      ColorPalette::WHITE, g_dropShadows);
 }
 
+KEEP_FUNC void GZ_frameCounter() {
+    if (!g_tools[FRAME_COUNT_INDEX].active) {
+        t = 0;
+        return;
+    }
+
+    if (GZ_getButtonTrig(GZPad::DPAD_UP)) {
+        if (!COUNT_STOP) {
+            COUNT_STOP = 1;
+        } else {
+            COUNT_STOP = 0;
+        }
+    }
+    char tm[10];
```

Comment:
this buffer might need to be bigger if there's no frame cap, otherwise it might overflow